citeproc/source/bibtex/bibtex.py

```diff
@@ -41,6 +41,9 @@ class BibTeX(BibliographySource):
 #              'year': None,
               'volume': 'volume',
 
+              'doi': 'DOI',
+              'url': 'URL',
```

Comment:
oh #66 tried to do the same, at least for url.
I myself yet to grasp ramification of a field being added, and since neither of PRs is accompanied with any tests to demonstrate that the change is taking an effect, I would be reluctant to merge.

Reply:
These map BibTeX fields to [CSL fields](http://docs.citationstyles.org/en/stable/specification.html#appendix-iv-variables).